server/src/infra/repositories/media.repository.ts

```diff
@@ -4,6 +4,7 @@ import ffmpeg, { FfprobeData } from 'fluent-ffmpeg';
 import sharp from 'sharp';
 import { promisify } from 'util';
 import fs from 'fs/promises';
+import { rgbaToThumbHash } from 'thumbhash-node';
```

Comment:
```suggestion
import { rgbaToThumbHash } from 'thumbhash';
```

Reply:
Yeah that's all good now. 
I had to do a reset because I messed up the lock.